File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_NgachLuongService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_NgachLuongService : BaseService<NS_DM_NgachLuong, INS_DM_NgachLuongRepository>
    , INS_DM_NgachLuongService
    {
    private readonly INS_DM_NgachLuongRepository _NS_DM_NgachLuongRepository;
    public NS_DM_NgachLuongService(INS_DM_NgachLuongRepository NS_DM_NgachLuongRepository) : base(NS_DM_NgachLuongRepository)
    {
    _NS_DM_NgachLuongRepository = NS_DM_NgachLuongRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_NgachLuong.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_NgachLuong } from './NS_DM_NgachLuong.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_NgachLuongService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaNgach', 'TenNgach', 'GhiChu', 'NhomNgach'];

    List: NS_DM_NgachLuong[] | undefined;
    ListFilter: NS_DM_NgachLuong[] | undefined;
    FormData!: NS_DM_NgachLuong;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_NgachLuong";
    }
}

